pkgs/applications/graphics/batik/default.nix

```diff
@@ -1,25 +1,19 @@
 {stdenv, fetchurl, unzip}:
 
-stdenv.mkDerivation {
-  name = "batik-1.6";
+stdenv.mkDerivation rec {
+  pname = "batik";
+  version = "1.13";
+
   builder = ./builder.sh;
```

Comment:
FYI I think this could probably just be an `installPhase` now. I suspect there are historical reasons why a builder script was used, but this style is kind of unusual and unnecessary for simple derivations like this.

If you feel like it please address it in this PR (as a separate commit, preferably). I don't think it should block this merge though, so feel free to push back.